react/Header/NewBadge/NewBadge.less

```diff
@@ -0,0 +1,17 @@
+@import (reference) "~seek-style-guide/theme";
+
+@height: 18px;
+
+.root {
+  content: 'New';
+  display: inline-block;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: @sk-medium;
+  color: @sk-white;
+  background-color: @sk-link;
```

Comment:
The contrast of this when on the active tab (or in the mobile menu) is quite low. Are we happy with that? (Given the text is still accessible)

Reply:
Yes. Navi has seen and approved this. Part of the reason is because we do not have the logic to make the label disappear after being interacted with so we lowered it's visual prominence for this iteration. It used to be pink haha